packages/webapp/src/inductions/components/induction-lists/induction-table.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { Heading } from "_app";
+
+export enum DataTypeEnum {
+    Action = "action",
+}
+
+export type Column = {
+    key: string;
+    label: string;
+    className?: string;
+    type?: DataTypeEnum;
+};
+
+export type Row = {
+    key: string;
+    [key: string]: string | JSX.Element;
+};
+
+interface InductionTableProps {
+    columns: Column[];
+    data: Row[];
+    className?: string;
+    headerClassName?: string;
+    rowClassName?: string;
+    tableHeader?: string;
+}
+
+export const Table = ({
```

Comment:
Can we move this component to `_app/ui`? I'd love to reuse this nice table across the board :P

Reply:
Yeah, you know, I think maybe it's generic enough to be more than an _Induction_ table. I'll do a quick pass over it, remove the word Induction and Ind here and there, and move it to `_app/ui`. 👍